modules/config-posture/main.tf

```diff
@@ -49,8 +47,11 @@ resource "sysdig_secure_cloud_auth_account_component" "oracle_service_principal"
   service_principal_metadata = jsonencode({
     oci = {
       api_key = {
-        user_id = local.sysdig_config_posture_user_ocid
+        user_id = data.sysdig_secure_trusted_oracle_app.config_posture.user_ocid
       }
+#       policy = {
+#         policy_id = oci_identity_policy.admit_onboarding_policy.id
+#       }
```

Comment:
Whenever uncommented, this will be `oci_identity_policy.admit_cspm_policy.id`.

Reply:
done!